src/Microsoft.Identity.Web/Constants/ClaimConstants.cs

```diff
@@ -99,5 +99,15 @@ public static class ClaimConstants
         /// Name Identifier ID claim: "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier".
         /// </summary>
         public const string NameIdentifierId = "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier";
+
+        /// <summary>
+        /// Optional claim used to determine the token type.
+        /// </summary>
+        public const string Idtyp = "idtyp";
+
+        /// <summary>
+        /// Optional claim used to determine if the user account is a guest account.
```

Comment:
we should not need this. There is corresponding WID claim. Check with Maxim.

Reply:
That wids claim is both undocumented and flaky.